x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A libpurple protocol plugin that adds support for the Telegram messenger"
+HOMEPAGE="https://github.com/majn/telegram-purple"
+SRC_URI="https://github.com/majn/telegram-purple/releases/download/v${PV}/telegram-purple_${PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+IUSE="+gcrypt +icons +nls +webp +zlib"
+
+RDEPEND="net-im/pidgin
+	gcrypt? ( dev-libs/libgcrypt:0= )
+	!gcrypt? ( dev-libs/openssl:0= )
+	nls? ( sys-devel/gettext )
+	webp? ( media-libs/libwebp )
+	zlib? ( sys-libs/zlib )"
+
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/telegram-purple"
+
+src_prepare() {
+	# Remove '-Werror' to make it compile
+	sed -i -e 's/-Werror //' Makefile.mingw || die
+	sed -i -e 's/-Werror //' Makefile.tgl.mingw || die
+	sed -i -e 's/-Werror //' tgl/Makefile.in || die
+	sed -i -e 's/-Werror //' tgl/tl-parser/Makefile.in || die
+
+	# Apply user patches
+	eapply_user
+}
+
+src_configure() {
+	# Run 'configure'
+	econf \
+		$(use_enable gcrypt) \
+		$(use_enable nls translation) \
+		$(use_enable webp libwebp) \
+		$(use_with zlib)
+}
+
+src_install() {
+	# Set docs
+	local DOCS=( "AUTHORS" "CHANGELOG.md" "HACKING.md" "HACKING.BUILD.md" "README.md" )
+
+	# Install icons depending on users choice
+	if ! use icons; then
+		# Run 'make install'
+		emake DESTDIR="${D}" noicon_install
+
+		# Install docs
+		einstalldocs
```

Comment:
Please get `einstalldocs` out of this condition. Documentation installation shouldn't depend on the `icons` USE flag.

Reply:
I am not sure, if this is really wrong? As I need to call noicon_install für USE="-icon", I am also calling einstalldocs. But if I use anyway USE="icon", I run default, which also calls einstalldocs? Or I am wrong? Anyway, I have put it outside of this block.